stdlib/LinearAlgebra/docs/src/index.md

```diff
@@ -269,6 +271,12 @@ Stacktrace:
 [...]
 ```
 
+If you need to solve many systems of the form `(A+μI)x = b`, it might be beneficial
```

Comment:
Perhaps clarify that this is for fixed `A` and different `μ` and `b` values (assuming I'm understanding correctly).